src/SamplesApp/UITests.Shared/Windows_Devices/AccelerometerTests.xaml.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Windows.Devices.Sensors;
+using Uno.UI.Samples.UITests.Helpers;
+using Windows.UI.Core;
+using Uno.UI.Samples.Controls;
+
+namespace UITests.Shared.Windows_Devices
+{
+	[SampleControlInfo("Windows.Devices", "Accelerometer", description: "Demonstrates use of Windows.Devices.Sensors.Accelerometer")]
+	public sealed partial class AccelerometerTests : UserControl
+	{
+
+		public AccelerometerTests()
+		{
+			this.InitializeComponent();
+			DataContext = new AccelerometerTestsViewModel(Dispatcher);
```

Comment:
Instead, you should add `typeof(AccelerometerTestsViewModel)` as 3rd parameter of the `SampleControlInfo` attribute. It will be created automatically by the sample navigation.